Update sdk

fraidev opened the pull request:
Commits:
- [x] Working with 1.4.0.
- [x] Working with main repository latest commit.
- [x] Using the custom AuthenticationQr again because the AuthenticationManager does not work well with custom login pages.
- [x] Getting the last commit
- [x] Fix newtonsoft version to fix ios build

Blocked:
- [x] We need the fix https://github.com/trilitech/tezos-unity-sdk/issues/77 to test the compiled build.

fraidev committed:
compile with 1.4.0 sdk

fraidev committed:
compile with main sdk

fraidev committed:
use custom AuthenticationQr again

fraidev committed:
get last commit

fraidev committed:
fix newtonsoft version to fix ios build

umutkutlu commented:
Tezos Unity SDK already includes the Newtonsoft.Json plugin for WebGL, [here.](https://github.com/trilitech/tezos-unity-sdk/blob/main/Runtime/Plugins/Newtonsoft.Json.dll) This is blocker for building the project to WebGL
![image](https://github.com/trilitech/tezos-unity-sdk-examples/assets/33513682/b5016254-c440-47c4-86cb-5227741a319b)


### Workaround
**Option 1:** Use the plugin that is already included in the SDK. (Preffered)
**Option 2:** Uncheck WebGL from yours.

fraidev committed:
remove newtosoft-json

fraidev commented:
> Tezos Unity SDK already includes the Newtonsoft.Json plugin for WebGL, [here.](https://github.com/trilitech/tezos-unity-sdk/blob/main/Runtime/Plugins/Newtonsoft.Json.dll) This is blocker for building the project to WebGL ![image](https://user-images.githubusercontent.com/33513682/244879078-b5016254-c440-47c4-86cb-5227741a319b.png)
> 
> ### Workaround
> **Option 1:** Use the plugin that is already included in the SDK. (Preffered) **Option 2:** Uncheck WebGL from yours.

I removed the dependency that I added before, and I created a [PR](https://github.com/trilitech/tezos-unity-sdk/issues/85) to add the plugin to IOS builds.
